crates/runtime/src/http/v1/mod.rs

```diff
@@ -52,6 +54,33 @@ pub enum Format {
     Csv,
 }
 
+#[derive(Default, Debug, Serialize, Deserialize)]
+#[serde(rename_all = "lowercase")]
+/// The various formats that the Arrow data can be converted and returned from HTTP requests.
+pub enum ArrowFormat {
+    #[default]
+    Json,
+    Csv,
+    Shell,
```

Comment:
```suggestion
    Text,
```

**nit:** `text/plain` is `Text`, not `Shell`?

Reply:
decided to go for `Plain` since Csv is `text/csv`